app/views/coronavirus_landing_page/components/landing_page/_statistic_section_block.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="covid-statistics__statistic-wrapper">
+  <%= render "govuk_publishing_components/components/heading", {
+      text: title,
+      heading_level: 3,
+      font_size: "s",
+      margin_bottom: 1
+    } %>
+
+  <p class="covid-statistics__statistic-value govuk-body">
+    <%= stat %>
+  </p>
+  <p class="govuk-body"><%= subtext %></p>
+
+  <p class="govuk-body">
+    Last updated <%= last_updated_date %>
+  </p>
```

Comment:
I feel like in Nikin's designs there might be a subtle reduction in font size for this date. It looks a little strange to me at the mo with the date as prominent as the copy. Not a biggie though, just something to check with him on.